doc/api/process.markdown

```diff
@@ -871,6 +881,39 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
+
+* `message` {Object}
+* `sendHandle` {Handle object}
+
+When node is spawned with an IPC channel attached, it can send messages to it's
+parent process using `process.send()`. They will be received as a
+['message'](child_process.html#child_process_event_message)
```

Comment:
@thefourtheye backticks aren't supported there, the markdown syntax I used is consistent with other places in docs where events are linked to.

@Trott thanks for all the comments, I will incorporate